spec/views/file_attachments/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe "file_attachments/edit.html.erb" do
+  describe "official documents" do
+    it "renders official document types" do
+      assign(:edition, build(:edition))
+      assign(:attachment, create(:file_attachment_revision))
+      render
+
+      types = %i[command_paper
+                 unnumbered_command_paper
+                 act_paper
+                 unnumbered_act_paper
+                 unofficial]
+
+      types.each do |type|
+        expect(rendered).to have_xpath("//input[@type='radio'][@value='#{type}']")
```

Comment:
it seems unusual to use xpath here it'd be more consistent to use a `have_selector("input[type='radio'][value='#{type}']")`

Reply:
Sure, fixed. Weirdly, I thought I tried that and it didn't work!